cmd/gardenlet/app/app.go

```diff
@@ -393,6 +394,12 @@ func (g *garden) Start(ctx context.Context) error {
 		return err
 	}
 
+	// TODO(shafeeqes): Remove this in gardener v1.90
```

Comment:
At least two releases would be better.
```suggestion
	// TODO(shafeeqes): Remove this in gardener v1.91
```